### PR TITLE
added ability to edit users from users page

### PR DESCRIPTION
**Pull request by danieljtate**

### CODE REVIEW

File: app/styles/css/users.css [hunk 1]
```diff
@@ -525,15 +588,143 @@ input[type=checkbox]:checked +label {
...
+  margin-left: 20px;
+}
+
+#userDetails td {
+  height: 40px;
+  width: 500px;
+  text-align: left;
+}
+
+#absences{
```
Comment (Michael-B-Lee) [inline app/styles/css/users.css]: Space before curly brace

File: app/components/Users.js [hunk 2]
```diff
@@ -298,16 +320,247 @@ var Users = React.createClass({
...
+      $('#absenceTable').slideUp("slow");
+      document.getElementById('absenceGlyphicon').src = "app/images/Arrow left - overlapping absences.png";
+    }
+  },
+
+userInfo: function (obj) {
+
+this.setState({
+  userInfo: obj
+
```
Comment (Michael-B-Lee) [inline app/components/Users.js]: remove line

File: app/components/Users.js [hunk 3]
```diff
@@ -298,16 +320,247 @@ var Users = React.createClass({
...
 
+    var allOfTheUsers = this.state.everyUser;
+    var approvers = [];
+    for (var i = 0; i < allOfTheUsers.length; i++){
+      if((allOfTheUsers[i].role === "APPROVER" || allOfTheUsers[i].role === "ADMIN") && (allOfTheUsers[i].name != document.getElementById("editName").value)){
+        approvers.push(allOfTheUsers[i]);
+      }
+    }
+
+
```
Comment (Michael-B-Lee) [inline app/components/Users.js]: spacing

File: app/components/Users.js [hunk 4]
```diff
@@ -365,6 +618,246 @@ var Users = React.createClass({
             </tbody>
           </table>
         </div>
+
+
+
+
```
Comment (AlexJYoungman) [inline app/components/Users.js]: too many new lines

File: app/components/Users.js [hunk 5]
```diff
@@ -944,6 +1438,8 @@ var UsersPage = React.createClass({
           </div>
         </div>
 
+
+
```
Comment (AlexJYoungman) [inline app/components/Users.js]: more unnecessary lines

**Pull request merged**